omero/sysadmins/unix/install-web.txt

```diff
@@ -114,16 +114,21 @@ OMERO.web Maintenance (Unix/Linux)
       :djangodoc:`Django clearing the session store documentation <topics/http/sessions/#clearing-the-session-store>`
       for more information.
 
-      .. note::
-          OMERO.web offers alternative session backends to automatically 
-          delete stale data using the cache session store backend, see
-          :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>`
-          for more details. After installing all the cache prerequisites set the following:
+   -  OMERO.web offers alternative session backends to automatically 
+      delete stale data using the cache session store backend, see
+      :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>`
+      for more details. After installing all the cache prerequisites set the following::
 
-          ::
+          $ bin/omero config set omero.web.session_engine django.contrib.sessions.backends.cache
 
-              $ bin/omero config set omero.web.caches '{"default": { "BACKEND": "django.core.cache.backends.memcached.MemcachedCache", "LOCATION": "127.0.0.1:11211", "TIMEOUT": "86400" } }'
-              $ bin/omero config set omero.web.session_engine django.contrib.sessions.backends.cache
+      - `Memcached <https://memcached.org/>`_::
+
+          $ bin/omero config set omero.web.caches '{"default": { "BACKEND": "django.core.cache.backends.memcached.MemcachedCache", "LOCATION": "127.0.0.1:11211", "TIMEOUT": "86400" } }'
```

Comment:
Understood, but in terms of what you put in place with WEBSESSIONS, it's not currently, right? And I assume we have no production servers using it. Mostly something to keep in mind, similar to the recent syslog addition, so that we don't have too many blocks of config that can age over time.